src/utilities/dockerUtils.ts

```diff
@@ -23,7 +23,12 @@ export async function getDeploymentConfig(): Promise<DeploymentConfig> {
       )
    }
 
-   const imageNames = core.getInput('images').split('\n') || []
+   let imageNames = core.getInput('images').split('\n') || []
+   imageNames.forEach((element) => {
+      if (element === null || '/^s+$/') {
```

Comment:
This looks like the correct regex but it's just a string. You aren't comparing it to element here.